extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/DebeziumCdcIntegrationTest.java

```diff
@@ -437,6 +438,42 @@ public void notFailWhenOldValueNotPresent() {
             assertTrueEventually(() -> assertThat(hz.getList("notFailWhenOldValueNotPresent")).isNotEmpty());
         }
     }
+    @Test
+    public void noFailWhenNoPrimaryKey() throws Exception {
+        try (PostgreSQLContainer<?> container = postgresContainer()) {
+            container.start();
+            //when
+            try (Connection connection = getPostgreSqlConnection(container.getJdbcUrl(), container.getUsername(),
+                    container.getPassword())) {
+                connection.setSchema("inventory");
+                Statement statement = connection.createStatement();
+                statement.addBatch("CREATE TABLE NO_PK (SOME_INT INT);");
+                statement.addBatch("INSERT INTO NO_PK VALUES (1)");
+                statement.executeBatch();
+            }
+
+            Pipeline pipeline = Pipeline.create();
+
+            StreamSource<ChangeRecord> source = DebeziumCdcSources
+                    .debezium("postgres", PostgresConnector.class)
+                    .setProperty("database.server.name", "dbserver1")
+                    .setProperty("database.hostname", container.getHost())
+                    .setProperty("database.port", Integer.toString(container.getMappedPort(POSTGRESQL_PORT)))
```

Comment:
You can use container.getFirstMappedPort() instead of container.getMappedPort()

Reply:
I can, but I think it's more explicit this way - I guess it's personal preference